images/oadp-operator.yml

```diff
@@ -0,0 +1,36 @@
+cachito:
+  packages:
+    gomod:
+      - path: .
+content:
+  source:
+    dockerfile: konflux.Dockerfile
+    git:
+      branch:
+        target: oadp-1.5
+      url: git@github.com:openshift-priv/oadp-operator.git
+      web: https://github.com/openshift/oadp-operator
+distgit:
+  component: oadp-operator-container
+delivery:
+  delivery_repo_names:
+  - oadp/oadp-rhel9-operator
+for_payload: false
+for_release: false
+enabled_repos:
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
+from:
+  builder:
+    - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
```

Comment:
Do we need to bring over `images/openshift-enterprise-base-rhel9.yml` from, for example, openshift-4.20 branch?

Reply:
ahh - I was operating under the initial assumption that we would be keeping it in one of the OCP branches. Sadly we can't bring that file over here

Instead we would need to add this to the streams.yml file of this branch: https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.19/streams.yml#L84

Reply:
Cool; already did it [here](https://github.com/openshift-eng/ocp-build-data/pull/7519/files#diff-9a621b2fa83a92ab54da125db08434c6f8362460c4085010a012ebee5a56299eR9)